projects/spectator/package.json

```diff
@@ -23,6 +23,7 @@
     "angular testing dumb components"
   ],
   "dependencies": {
+    "@angular-devkit/core": ">= 10.1.0",
```

Comment:
Why do we need it?

Reply:
Wenn I tried to run this as a linked package it complained that @angular-devkit/core was not found although it was installed in the project.